.circleci/config.yml

```
@@ -25,9 +25,26 @@ jobs:
           path: target/daml-test-reports
       - store_artifacts:
           path: target/daml-test-reports
+          
+  npm_test:       
+    executor: daml-executor
```

after adding a node executor
```
executors:
  daml-executor:
    docker:
      - image: digitalasset/daml-sdk:0.12.15-circleci-master
  node-executor:
    docker:
      - image: node:8.12-alpine
```
then

```suggestion
    executor: node-executor
```

so we only need npm install below